### PR TITLE
Added unit test for loading GeoDataFrames into workspace

### PR DESCRIPTION
**Pull request by dzelge**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 * Fixed code signing issue during the installer build on MacOS and Windows [#726](https://github.com/CCI-Tools/cate/issues/726)
 * Fixed Cate Desktop failed to start (in Ubuntu 18) due to missing .so [#729](https://github.com/CCI-Tools/cate/issues/729)
 * Fixed Cate Desktop failed to start (in Windows) due to unable to find dll files [#725](https://github.com/CCI-Tools/cate/issues/725)
+* Added unit test for loading GeoDataFrames into workspace
```
[the remark this answers is not in the document]
Comment (dzelge) [reply CHANGES.md]: Done

File: CHANGES.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 * Fixed code signing issue during the installer build on MacOS and Windows [#726](https://github.com/CCI-Tools/cate/issues/726)
 * Fixed Cate Desktop failed to start (in Ubuntu 18) due to missing .so [#729](https://github.com/CCI-Tools/cate/issues/729)
 * Fixed Cate Desktop failed to start (in Windows) due to unable to find dll files [#725](https://github.com/CCI-Tools/cate/issues/725)
+* Added unit test for loading GeoDataFrames into workspace [#705](https://github.com/CCI-Tools/cate/issues/705)
```
Comment (forman) [inline CHANGES.md]: As said before, unit-test are "nominal", no need to mention them here.

File: CHANGES.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
...
+* Cate Desktop: Added information about GeoDataFrames in the details section [#705](https://github.com/CCI-Tools/cate/issues/705)
```
Comment (forman) [inline CHANGES.md]: 1 issue = 1 line in here, no need to mention same issue twice

File: cate/core/workspace.py [hunk 4]
```diff
@@ -375,7 +372,7 @@ def _update_resource_json_from_feature_collection(cls, resource_json, features:
             'crs_wkt': crs_wkt,
             'numFeatures': num_features,
         }
-
+        print('Hello')
```
Comment (forman) [inline cate/core/workspace.py]: Remove please :)

File: cate/core/workspace.py [hunk 5]
```diff
@@ -362,6 +358,7 @@ def _update_resource_json_from_feature_collection(cls, resource_json, features:
                     'dataType': var_type,
                     'isFeatureAttribute': True,
                 })
+
         geometry = features.schema.get('geometry')
         crs = str(features.crs)
         crs_wkt = str(features.crs_wkt)
```
Comment (forman) [inline cate/core/workspace.py]: Use name `crsWkt`

**Pull request merged**